### PR TITLE
Add transform all

### PR DESCRIPTION
**Pull request by nleanba**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -33,7 +33,14 @@ const WEBHOOK_SECRET: string | undefined = Deno.env.get("WEBHOOK_SECRET");
 await Deno.mkdir(`${config.workDir}/repo`, { recursive: true });
 await Deno.mkdir(`${config.workDir}/tmprdf`, { recursive: true });
 await Deno.mkdir(`${config.workDir}/tmpttl`, { recursive: true });
-await createBadge("Unknown");
+
+const db = new JobsDataBase(`${config.workDir}/jobs`);
```
Comment (retog) [inline src/main.ts:37]: Why not take care of the badge within the worker?

File: src/action_worker.ts [hunk 2]
```diff
@@ -20,10 +20,14 @@ let isRunning = false;
...
-  const job = evt.data as Job;
-  queue.addJob(job);
-  if (!isRunning) startTask();
-  else console.log("Already running");
+  const job = evt.data as Job | "FULLUPDATE";
+  if (job === "FULLUPDATE") {
+    gatherJobsForFullUpdate();
+  } else {
+    queue.addJob(job);
+    if (!isRunning) startTask();
```
Comment (retog) [inline src/action_worker.ts:28]: shouldn't this be called outside the else? (i.e in any case)

Comment (retog) [reply src/action_worker.ts:28]: Okay, I see what you're doing. I'm a bit skeptical about the re-use of `isRunning`. Meditating on it.

File: src/main.ts (hunk 1 above)
Comment (nleanba) [reply src/main.ts:37]: badge is made by the worker, this is only on initial startup

File: src/action_worker.ts (hunk 2 above)
Comment (nleanba) [reply src/action_worker.ts:28]: `startTask()` is called  by `gatherJobsForFullUpdate()`, this way the message handler doesn't need to await `gatherJobsForFullUpdate()`.

`isRunning` blocks jobs from running while gathering, to ensure that the full update runs on a consistent initial state.

**Pull request merged**